packages/components/src/RichTextEditor/RichTextEditor/RichTextEditor.tsx

```diff
@@ -144,7 +145,11 @@ export const RichTextEditor = ({
     : ""
   const descriptionAria = description ? `${editorId}-rte-description` : ""
 
-  const ariaDescribedBy = `${validationMessageAria} ${descriptionAria}`
+  const ariaDescribedBy = classnames(
```

Comment:
Why the use of `classnames` here?

Reply:
Clean way to concat these strings with a space in between

Reply:
Compared to string interpolation, if one of them is falsy, it won't add the extra space or the word `undefined` which happens sometimes.

Reply:
<img width="1698" alt="image" src="https://github.com/cultureamp/kaizen-design-system/assets/1811583/2db809b3-49a6-4f87-9682-44aff99fc7bb">

Reply:
Oh, that's a creative use, but I don't think its very clear. Maybe add a comment or find a utility that is a bit more closer to the use?

Reply:
Respectfully disagree because this is the one thing that the function does. Wanna put in a review suggestion for what you think the comment should be?

Reply:
Hm fair enough